content/topics/messaging/bridging-course-intro/_index.md

```diff
@@ -41,28 +41,28 @@ If you can UNDERSTAND all the stuff covered in this course then you're golden. *
+If you submitted work during the Bootcamp and you didn't 100% understand it, then please go back and revisit that stuff.  Re-do the tutorials, practise as much as you need to, and make sure you master the material. You should be able to do the projects yourself!
 
 As a professional technologist you will be asked to do way harder stuff, **if you want a career in tech, then master the material**.
 
 ## It's our job to make content that helps you understand
 
 That means that if anything is ambiguous or confusing, we need to sort things out so that it makes sense.
 
-We wont be spoonfeeding you though. We want to turn you into a professional. Professionals overcome problems and seek understanding. So if you ask a question that you should be able to figure out on your own we might tell you to go figure it out. Don't take it personally.
+We won't be spoonfeeding you though. We want to turn you into a professional. Professionals overcome problems and seek understanding. So if you ask a question that you should be able to figure out on your own we might tell you to go figure it out. Don't take it personally.
 
-**Seeking understanding is like gym for your brain.**
+**Seeking understanding is like a gym for your brain.**
 
-## This is self paced learning
+## This is self-paced learning
 
 Cast your mind back to when you were in school. If you went to a normal school then you would have been in a class of kids who were all forced to work at the same pace. This was really bad for a lot of people.
 
-Many of the faster kids were stunted and board instead of stretched. Opportunities for growth were lost.
-And many of the slower kids just gave up and decided that they were incapable. A lot of potential simply got thrown away.
+Many of the "faster" kids were stunted and bored instead of stretched. Growth opportunities were lost.
+Many of the "slower" kids just gave up and decided that they were incapable. A lot of potential simply got thrown away.
 
-Now, this is a remote course. Different people doing our course have different interests, aspirations and home lived. We believe that making everyone work in lock-step would be really a silly thing to do. So you get to work at your own pace.
+Now, this is a remote course. Different people doing our course have different interests, aspirations and home lives. We believe that making everyone work in lockstep would be a silly thing to do. So you get to work at your own pace.
 
-We do, however, have standards: We do have some minimum pace requirements you'll need to meet, otherwise you just wont learn enough to gain a useful set of skills. If your pace is too slow we will let you know.
+We do, however, have standards: We do have some minimum "pace" requirements you'll need to meet. Otherwise, you just won't learn enough to gain a useful set of skills. If your pace is too slow we will let you know.
```

Comment:
```suggestion
We do, however, have standards: We do have some minimum "pace" requirements you'll need to meet. Otherwise, you just won't learn enough to gain a useful set of skills. If your pace is too slow, we will let you know.
```